### PR TITLE
Menu + settings redesign UI changes

### PR DESCRIPTION
**Pull request by Bilb**

### CODE REVIEW

File: ts/components/conversation/message/message-content/MessageContentWithStatus.tsx [hunk 1]
```diff
@@ -122,11 +126,11 @@ export const MessageContentWithStatuses = (props: Props) => {
           messageId={messageId}
           isCorrectSide={isIncoming}
         />
-        <div>
+        <StyledMessageWithAuthor isOutgoing={!isIncoming}>
```
Comment (yougotwill) [inline ts/components/conversation/message/message-content/MessageContentWithStatus.tsx:129]: Could we use isIncoming like we do for other styled components?

**Review by yougotwill — CHANGES_REQUESTED** [review]

File: ts/components/conversation/message/message-content/MessageContentWithStatus.tsx (hunk 1 above)
Comment (Bilb) [reply ts/components/conversation/message/message-content/MessageContentWithStatus.tsx:129]: ok

**Review by yougotwill — APPROVED** [review]
LGTM 🚀

File: stylesheets/_session_password.scss [hunk 2]
```diff
@@ -33,7 +33,7 @@
     &-error-section {
       width: 100%;
       color: $session-color-white;
-      margin: -var(--margins-sm) 0px 2 * $session-margin-lg 0px;
+      margin: 10px;
```
Comment (yougotwill) [inline stylesheets/_session_password.scss:36]: Shouldn't you use var(--margins-sm) here instead of 10px?

File: ts/mains/password_start.tsx [hunk 3]
```diff
@@ -3,5 +3,11 @@
 
 import React from 'react';
 import { SessionPasswordPrompt } from '../components/SessionPasswordPrompt';
+import { SessionTheme } from '../state/ducks/SessionTheme';
 
-window.ReactDOM.render(<SessionPasswordPrompt />, document.getElementById('root'));
+window.ReactDOM.render(
+  <SessionTheme >
+    <SessionPasswordPrompt  />
```
Comment (yougotwill) [inline ts/mains/password_start.tsx:10]: Extra space here

**Review by yougotwill — CHANGES_REQUESTED** [review]

File: ts/mains/password_start.tsx (hunk 3 above)
Comment (Bilb) [reply ts/mains/password_start.tsx:10]: fixed

File: stylesheets/_session_password.scss (hunk 2 above)
Comment (Bilb) [reply stylesheets/_session_password.scss]: yeah sorry I made that change before i wrapped the whole thing in `SessionTheme`. Now that SessionTheme is surrounding (and call the switch theme on mount) we have those values available

**Review by yougotwill — APPROVED** [review]
LGTM!

**Pull request merged**